src/gui/CemuApp.cpp

```diff
@@ -218,10 +218,6 @@ int CemuApp::FilterEvent(wxEvent& event)
 		const auto& key_event = (wxKeyEvent&)event;
 		g_window_info.set_keystate(fix_raw_keycode(key_event.GetRawKeyCode(), key_event.GetRawKeyFlags()), false);
 	}
-	else if(event.GetEventType() == wxEVT_KILL_FOCUS)
-	{
-		g_window_info.set_keystatesdown();
-	}
```

Comment:
On linux pressing tab triggers this event, immediately resetting the tab key state. Makes the button unpressable. Perhaps some workaround could work where ignores the kill focus events when tab is held.

Reply:
Fixed by checking if window is active in LatterRenderTarget. Perhaps it would be nicer to just set all keys to up when app is marked inactive in CemuApp. That way the Keyboard input manager also doesn't need to look that up.